deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,349 @@
+```sql
+psql -U u5dh6v2ymn********** -d bjzfhkl5qc**********
+
+bjzfhkl5qc**********=> show pool_pools;
+ pool_pid |     start_time      | pool_id | backend_id |       database       |       username       |     create_time     | majorversion | minorversion | pool_counter | pool_backendpid | pool_connected
+----------+---------------------+---------+------------+----------------------+----------------------+---------------------+--------------+--------------+--------------+-----------------+----------------
+ 3526     | 2021-06-02 15:22:12 | 0       | 0          | bjzfhkl5qc********** | u5dh6v2ymn********** | 2021-06-02 15:22:35 | 3            | 0            | 1            | 1520665         | 1
+ 3526     | 2021-06-02 15:22:12 | 0       | 1          | bjzfhkl5qc********** | u5dh6v2ymn********** | 2021-06-02 15:22:35 | 3            | 0            | 1            | 1665071         | 1
+ 3526     | 2021-06-02 15:22:12 | 0       | 2          | bjzfhkl5qc********** | u5dh6v2ymn********** | 2021-06-02 15:22:35 | 3            | 0            | 1            | 531483          | 1
+ 3527     | 2021-06-02 15:22:12 | 0       | 0          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ 3527     | 2021-06-02 15:22:12 | 0       | 1          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ 3527     | 2021-06-02 15:22:12 | 0       | 2          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ 3528     | 2021-06-02 15:22:12 | 0       | 0          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ 3528     | 2021-06-02 15:22:12 | 0       | 1          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ 3528     | 2021-06-02 15:22:12 | 0       | 2          |                      |                      |                     | 0            | 0            | 0            | 0               | 0
+ ...      | ...                 | .       | .          |                      |                      |                     | .            | .            | .            | .               | .
+```
+
+We can see the 32 available processes (**3526**, **3527**, **3528...**), and active connections for process **3526**, with an open connection for each postgresql server (**1520665**, **1665071** and **531483**).
+
+If a process remains inactive for more than **300** seconds, it will be deleted and a new process will take its place. This value can be configured with the `CC_PGPOOL_CHILD_LIFE_TIME` variable. This is a measure to prevent the memory leaks and other unexpected errors in Pgpool-II children.
```

Comment:
```suggestion
If a process remains inactive for more than **300** seconds, it will be deleted and a new process will take its place. This value can be configured with the `CC_PGPOOL_CHILD_LIFE_TIME` variable. This is a measure to prevent memory leaks and other unexpected errors in Pgpool-II children.
```